src/components/menu-item/assets/menu-item/t9n/messages.json

```diff
@@ -0,0 +1,4 @@
+{
+  "submenu": "submenu",
```

Comment:
Should these be capitalized (like other message bundles)?

Reply:
Good Catch, the value should be capitalized.